services/ui-src/src/components/fields/DynamicField.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+//components
+import { DynamicField } from "components";
+
+const appendButtonText = "Add a row";
```

Comment:
Nit: If you're only using this once, I'd argue it doesn't need to be defined globally -- just use the string in the test.

Reply:
thought it might be easier to update later if the text in dynamic form changes. Can do locally though